webui/src/lib/components/auth/forms.jsx

```diff
@@ -78,6 +87,11 @@ export const AttachModal = ({
       </>
     );
 
+  const handleSearchChange = () => {
+      setSearchPrefix(search.current.value);
```

Comment:
Note that this invokes two renders.
I highly suggest creating a single state object for the pagination params (`{prefix, after}`),
And using it instead of these two state params.